allauth/account/app_settings.py

```diff
@@ -386,6 +386,10 @@ def PASSWORD_RESET_TOKEN_GENERATOR(self):
             token_generator = EmailAwarePasswordResetTokenGenerator
         return token_generator
 
+    @property
+    def SEND_TO_UNKNOWN_EMAILS(self):
+        return self._setting("SEND_TO_UNKNOWN_EMAILS", True)
```

Comment:
So I started out with `ACCOUNTS` here and then I didn't like it because we're specifically referring to email addresses which are not accounts.
How would you feel about `EMAIL_UNKNOWN_ADDRESSES`?

Reply:
You're right about using `ACCOUNTS`, though on thing that is has going for it is that the email that this setting disabled is this one:

``` python
        get_adapter().send_mail("account/email/unknown_account", email, context)
```

So naming wise it is consistent with that...

Reply:
@pennersr in the name of getting this merged in any form, I'll just accept this :)